package.json

```diff
@@ -56,18 +56,31 @@
     "commands": [
       {
         "command": "docs.signIn",
-        "title": "Sign-in to Docs portal",
+        "title": "Sign-in to Docs build system",
         "category": "Docs"
       },
       {
         "command": "docs.signOut",
-        "title": "Sign-out from Docs portal",
+        "title": "Sign-out from Docs build system",
         "category": "Docs"
       },
       {
         "command": "docs.build",
         "title": "Build this workspace folder",
         "category": "Docs"
+      },
+      {
+        "command": "docs.validationQuickPick",
+        "title": "Docs: validation menu",
+        "category": "Docs"
+      }
+    ],
+    "keybindings": [
+      {
+        "command": "docs.validationQuickPick",
+        "key": "Alt+D",
+        "mac": "Alt+D",
```

Comment:
I think we don't need hotkey for now. In case it will conflict with other plugins

Reply:
it is the feature design from PM, we can confirm with Curt Wednesday

Reply:
👍keep it now, I'm OK with it but just curious about it.

---
In reply to: [373910024](https://github.com/docascode/vscode-docs-build/pull/33#discussion_r373910024) [](ancestors = 373910024)